src/card/stories.js

```diff
@@ -0,0 +1,153 @@
+/* @flow */
+/*global module */
+import React from 'react';
+
+import {storiesOf} from '@storybook/react';
+import {withReadme} from 'storybook-readme';
+
+//$FlowFixMe
+import CardReadme from '../../rfcs/card-component.md';
+
+import {mergeOverrides} from '../helpers/overrides';
+
+import {
+  Card as BaseCard,
+  StyledAction,
+  StyledBody,
+  StyledThumbnail,
+  StyledTitle,
+  hasThumbnail,
+} from './';
+
+import {thumbnail as thumbnailImg, header as headerImg} from './images';
+
+import type {CardsPropsT} from './types';
+
+const cardWidth = '328px';
+const placeholderTitle = 'Card Title Entry';
+const placeholderText =
+  'Proin ut dui sed metus pharetra hend rerit vel non mi. Nulla ornare faucibus ex, non facilisis nisl. Maecenas aliquet mauris ut tempus cursus. Etiam semper luctus sem ac blandit. ';
+
+const Card = (props: CardsPropsT) => {
+  const {overrides, ...otherProps} = props;
+  const cardOverrides = mergeOverrides(
+    {
+      Root: {
+        style: {
+          width: cardWidth,
+        },
+      },
+    },
+    overrides,
+  );
+  //$FlowFixMe
+  return <BaseCard {...otherProps} overrides={cardOverrides} />;
+};
+Card.defaultProps = {
+  hasThumbnail,
+  overrides: {},
+};
+
+function Button(props) {
+  return <button {...props} />;
+}
+
+storiesOf('Card', module)
+  .addDecorator(withReadme(CardReadme))
+  .add('Headline & Text', () => (
+    <Card>
+      <StyledTitle>{placeholderTitle}</StyledTitle>
+      <StyledBody>{placeholderText}</StyledBody>
+    </Card>
+  ))
+  .add('Text only', () => (
+    <Card style={{width: cardWidth}}>
```

Comment:
no need for passing this style anymore since it's handled in the `Card` custom component

Reply:
good catch